lib/routes/bankless/article.ts

```diff
@@ -0,0 +1,56 @@
+import { Route } from '@/types';
+import ofetch from '@/utils/ofetch';
+import cache from '@/utils/cache';
+import { load } from 'cheerio';
+import { parseDate } from '@/utils/parse-date';
+const baseUrl = 'https://www.bankless.com/read';
+export const route: Route = {
+    path: '/article',
+    categories: ['finance'],
+    example: '/bankless/article',
+    radar: [
+        {
+            source: ['bankless.com/read'],
+        },
+    ],
+    name: 'Bankless articles',
+    maintainers: ['Colin'],
+    handler,
+};
+async function handler() {
+    const response = await ofetch(baseUrl);
```

Comment:
Please use the API endpoint `/api/read/filter-content` instead

Reply:
There is no needs to provide the api, only /read page is useful.

Reply:
If a site uses its API to provide articles, the feed should prioritize the API instead of scraping the site's HTML

Reply:
> If a site uses its API to provide articles, the feed should prioritize the API instead of scraping the site's HTML

Where can find the site's api?I didn't see it.

Reply:
Oh, I saw it. But the api need authencation and api key, I think it's not easy for users to use it, right?

Reply:
Visit https://github.com/DIYgod/RSSHub/blob/ef8c08d2fc60173e4302b3a5ae8006f38c62d44c/lib/routes/bankless/article.ts#L6
and search `/api/read/filter-content` in the DevTools

Reply:
> Visit
> 
> https://github.com/DIYgod/RSSHub/blob/ef8c08d2fc60173e4302b3a5ae8006f38c62d44c/lib/routes/bankless/article.ts#L6
> 
> 
> and search `/api/read/filter-content` in the DevTools

Ok, I see. But it will take me much more time to deal with the filter type, can you merge the current route which with the whole first 40 articles at first?

Reply:
Switching to API does not require much effort, and it's expected to get it fixed before merging